docs/en/developers/03_Backend/05_Extensions.md

```diff
@@ -347,6 +347,9 @@ The following events are available:
 * `nav_reading_modes` (`function($reading_modes) -> array | null`): **TODO** add documentation.
 * `post_update` (`function(none) -> none`): **TODO** add documentation.
 * `simplepie_before_init` (`function($simplePie, $feed) -> none`): **TODO** add documentation.
+* `entry_favorite` (`function(array($id, $is_favorite)) -> true | false | null`): will be executed, when the favorite status of an entry is changed
```

Comment:
Is there a specific thought behind adding them here or is it just added to the bottom? Because in the latter case, it probably makes more sense near the top instead.

Reply:
Sorry, of course you are right, I've just added them there with no reason. Changed it to an alphabetical order.